ui/app/AppLayouts/Wallet/views/RightTabView.qml

```diff
@@ -178,7 +180,7 @@ RightTabBaseView {
                         filterVisible: filterButton.checked
                         onCollectibleClicked: {
                             RootStore.collectiblesStore.getDetailedCollectible(chainId, contractAddress, tokenId)
-                            RootStore.setCurrentViewedHolding(uid, tokenType)
+                            RootStore.setCurrentViewedHolding(uid, uid, tokenType)
```

Comment:
Intended? (double uid)

Reply:
yes Ive added this tokenKeys param temporarily. Send and Bridge are using symbol instead of tokensKey, and that needs to be updated but I dont want to break those flows right now. And hence I added a new param and in case of collectibles its the same.